specification/paths/ExportLoginAttempts.json

```diff
@@ -6,7 +6,8 @@
     "security": [
       {
         "OAuth2": [
-          "experimental"
+          "experimental",
+          "logs.view"
```

Comment:
- Let's list this scope on top of the spec with the other scopes.
- Do we still need the experimental scope on this endpoint?

Reply:
I suggest we keep the experimental scope so that it is only used by the portal.

Reply:
On second thought - no, we do not really need it. I will remove it